sklearn/linear_model/ridge.py

```diff
@@ -1089,7 +1096,7 @@ def __init__(self, alphas=(0.1, 1.0, 10.0),
                  fit_intercept=True, normalize=False, scoring=None,
                  cv=None, gcv_mode=None,
                  store_cv_values=False):
-        self.alphas = alphas
+        self.alphas = np.asarray(alphas, dtype=np.float64)
```

Comment:
Usually we do not alter parameters in `__init__`, because they can also be set in other ways. We delay all validation until fit (except in old code)

Reply:
my mistake, an error of the file. changed again